CHANGELOG.md

```diff
@@ -2,7 +2,9 @@
 
 ##### Breaking
 
-* None.
+* `file_length` rule now has a default value of `120` for warnings.  
```

Comment:
Though not really a breaking change, I do approve of having it in this section to spread the word...